docs/source/development-environment.rst

```diff
@@ -0,0 +1,141 @@
+Zipline Development Guidelines
+==============================
+
+Environment Setup
+-----------------
+
+First, you'll need to clone zipline by running:
+
+.. code-block:: bash
+
+   git clone git@github.com:quantopian/zipline.git
+
+Then check out to a new branch where you can make your changes:
+
+.. code-block:: bash
+		
+   git checkout -b some-short-descriptive-name
+
+The following section assumes you already have virtualenvwrapper and pip installed on your system. If you don't already have them, you'll need some C library dependencies. On Linux you can run:
+
+.. code-block:: bash
+
+   # on linux
+   $ sudo apt-get install libopenblas-dev liblapack-dev gfortran
```

Comment:
Agreed on this.

Also, FWIW, `apt-get` is specific to linux distributions like Ubuntu that use Debian packages (c.f. https://wiki.debian.org/Apt, https://wiki.debian.org/Packaging/BinaryPackage).  Other linux distributions use different package managers.  For example, Fedora uses [`dnf`](https://fedoraproject.org/wiki/Package_management_system), and Arch uses [pacman](https://wiki.archlinux.org/index.php/pacman).